client/my-sites/media/main.jsx

```diff
@@ -167,13 +165,21 @@ class Media extends Component {
 			return null;
 		}
 
-		const { translate } = this.props;
+		const { selectedSite, translate } = this.props;
+		const selectedMediaCount = MediaLibrarySelectedStore.getAll( selectedSite.ID ).length;
 
 		return [
 			{
 				action: 'delete',
 				additionalClassNames: 'is-borderless is-scary',
-				label: translate( 'Delete' ),
+				label:
+					selectedMediaCount < 2
+						? translate( 'Delete item' )
```

Comment:
Note that `translate` has [pluralization support](https://github.com/Automattic/wp-calypso/tree/master/packages/i18n-calypso#pluralization), so this could be  changed to:

```js
// eslint-disable-next-line wpcalypso/i18n-mismatched-placeholders
label: translate(
		'Delete item',
		'Delete %(selectedMediaCount)s items',
		{
				count: selectedMediaCount,
				args: { selectedMediaCount },
		}
),
```